framework/OutStreams/PlotInterfaces/PopulationPlot.py

```diff
@@ -0,0 +1,166 @@
+# Copyright 2017 Battelle Energy Alliance, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Created on November 20th, 2021
+
+@author: mandd
+"""
+
+# External Imports
+from collections import defaultdict
+import matplotlib.pyplot as plt
+import numpy as np
+from matplotlib.lines import Line2D
+from .PlotInterface import PlotInterface
+from utils import InputData, InputTypes
```

Comment:
Could you move these lines to the "Internal Imports"?

Reply:
moved